README.md

```diff
@@ -32,7 +32,7 @@ If your collection is not present on the Ansible forum yet, please check out the
 
 * Join the Ansible forum:
     * [Get Help](https://forum.ansible.com/c/help/6): get help or help others.
```

Comment:
How about also mentioning the tag in this line?
```suggestion
    * [Get Help](https://forum.ansible.com/c/help/6): get help or help others. Please add appropriate tags if you start new discussions, for example the `YOUR TAG` tag.
```

Reply:
LGTM, feel free to raise the same suggesting in https://github.com/ansible-community/project-template/pull/16 (or i can later if you have no time)